source/_integrations/awtrix3.markdown

```
@@ -0,0 +1,139 @@
+  type: string
+data: 
+  description: specific data for message
+  required: false
+  type: map	
+  keys:
+    PARAMETER_NAME:
+      description: The value of the variable. Any YAML is valid. Templates can also be used to pass a value to the variable.
+      type: any
+{% endconfiguration %}
+
+## Example
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)